### PR TITLE
Timeline  Plugin - Table Sorting

### PR DESCRIPTION
**Pull request by DevGregor**
This makes the table of the timeline plugin sortable.

### CODE REVIEW

File: plugins/timeline.js [hunk 1]
```diff
@@ -115,14 +114,145 @@ function createContent(data, pluginHelper) {
...
 
     return popupContentPrefix + `</tbody></table>`;
 }
 
-pluginList.push(plugin);
\ No newline at end of file
+pluginList.push(plugin);
+
+/*
+	A simple, lightweight jQuery plugin for creating sortable tables.
```
Comment (incpi) [inline plugins/timeline.js:128]: Hello,

This seems to be the wrong place where you add this library.

It should go to the lib/scematic folder. So it should be accessible everywhere and maintain property. As maybe tablesort.js or something suitable.

 
Thanks,
Omkar

**Review by incpi — CHANGES_REQUESTED** [review]

File: plugins/timeline.js (hunk 1 above)
Comment (DevGregor) [reply plugins/timeline.js:128]: Hello Omkar

Yes, this script is from Fomantic. I wasn't sure where to put it so I put it together with the plugin where I needed it.

Comment (incpi) [reply plugins/timeline.js]: Hello,

FYI,You can also use icons [SAP web UI](https://dbeck121.github.io/CPI-Helper-Chrome-Extension/?page=development) for lock, unlock similar to inline debug mode.

No change requested. Thanks a lot.

**Comment by incpi** [comment]
Hello,

FYI,You can also use icons [SAP web UI](https://dbeck121.github.io/CPI-Helper-Chrome-Extension/?page=development) for lock, unlock similar to inline debug mode.

No change requested. Thanks a lot.

**Comment by DevGregor** [comment]
Tried to make a separate draft Pull request but I don't know how or it either doesn't work.

**Comment by dbeck121** [comment]
No worries :-)

I pulled your changes and just in case you want to make changes, create another one.

Thanks for the contribution. I like your plugin. Will be released with next version.

**Pull request merged**